src/xrCore/Threading/ThreadUtil.cpp

```diff
@@ -11,10 +11,9 @@ ThreadId GetCurrThreadId() { return GetCurrentThreadId(); }
 
 ThreadHandle GetCurrentThreadHandle() { return GetCurrentThread(); }
 
-void SetThreadName(ThreadHandle threadHandle, pcstr name)
+void SetThreadNameImpl(DWORD threadId, pcstr name)
```

Comment:
What is the reason behind replacing `ThreadHandle` by `DWORD`? The last is not cross platform as I know.

Reply:
Nothing being replaced, it's just implementation moved in its own function, `SetThreadName` and `SetCurrentThreadName` still has `ThreadHandle` as first parameter type.